opencog/query/PatternSCM.cc

```diff
@@ -69,11 +69,17 @@ void PatternSCM::init(void)
 	// BindLink containing variables, a pattern and a rewrite rules.
 	_binders.push_back(new FunctionWrap(bindlink, "cog-bind", "query"));
 
-	// Identical to do_bindlink above, except that it only returns the
+	// Identical to bindlink above, except that it only returns the
 	// first match.
 	_binders.push_back(new FunctionWrap(single_bindlink,
 	                   "cog-bind-single", "query"));
```

Comment:
as you mention elsewhere, the code for this should be ripped out and replaced by the one-liner
```
(define-public (cog-bind-single atom) (cog-bind-first-n 1 atom))
```
in query.scm